lambdas/db-migration/src/migrations/20200901060127_create_files.ts

```diff
@@ -0,0 +1,23 @@
+import * as Knex from 'knex';
+
+export const up = async (knex: Knex): Promise<void> => knex.schema
+  .createTable('files', (table) => {
+    table.bigIncrements('cumulusId').comment('Internal Cumulus ID for a file').primary();
+    table.integer('granuleCumulusId').references('cumulusId').inTable('granules').notNullable();
+    table.timestamps(false, true);
+    table.integer('fileSize').comment('Deprecated - size of file');
```

Comment:
If this field is deprecated, do we need to be creating it in the DB?

Reply:
I explicitly remember the conversation here being that we don't want to remove any fields to avoid complication in the migration